sortdb/sortdb_metrics.json

```diff
@@ -0,0 +1,30 @@
+{
+  "sortdb.stats.total_requests": 2,
+  "sortdb.stats.total_seeks": 24,
+  "sortdb.stats.get_requests": 3,
+  "sortdb.stats.get_hits": 3,
+  "sortdb.stats.get_misses": 0,
+  "sortdb.stats.get_requests.avg": 448,
+  "sortdb.stats.get_requests.95percentile": 1323,
+  "sortdb.stats.get_requests.99percentile": 1323,
+  "sortdb.stats.mget_requests": 0,
+  "sortdb.stats.mget_hits": 0,
+  "sortdb.stats.mget_misses": 0,
+  "sortdb.stats.mget_requests.avg": 0,
+  "sortdb.stats.mget_requests.95percentile": 0,
+  "sortdb.stats.mget_requests.99percentile": 0,
+  "sortdb.stats.fwmatch_requests": 1,
+  "sortdb.stats.fwmatch_hits": 1,
+  "sortdb.stats.fwmatch_misses": 0,
+  "sortdb.stats.fwmatch_requests.avg": 10,
+  "sortdb.stats.fwmatch_requests.95percentile": 10,
+  "sortdb.stats.fwmatch_requests.99percentile": 10,
+  "sortdb.stats.range_requests": 2,
+  "sortdb.stats.range_hits": 1,
+  "sortdb.stats.range_misses": 1,
+  "sortdb.stats.range_requests.avg": 18,
+  "sortdb.stats.range_requests.95percentile": 24,
+  "sortdb.stats.range_requests.99percentile": 24,
+  "sortdb.stats.db_size.bytes": 767557632,
+  "sortdb.stats.db_mtime": 1435463934
+}
```

Comment:
Does this file get used for anywhere?

Reply:
This file is used as input in test_sortdb.py for mock tests.

Reply:
You have 2 files here with mock data, and it looks like the test is using `sortdb/ci/sortdb_metrics.json` instead of this one: 
https://github.com/namrata4/integrations-extras/blob/c2365267473006ed04b261c495d37c6b3ce603de/sortdb/test_sortdb.py#L59
If you could verify this and remove the extra file, that would be great